extensions/binding/org.eclipse.smarthome.binding.mqtt/README.md

```diff
@@ -0,0 +1,97 @@
+# MQTT Broker Connections
+
+    MQTT is a machine-to-machine (M2M)/"Internet of Things" connectivity protocol.
+    It was designed as an extremely lightweight publish/subscribe messaging transport.
+
+This binding allows to list non-mutable system MQTT broker connections for status check purposes as well as configure new broker connections.
```

Comment:
Make more generic to describe the mqtt binding as such (plus explain the system broker features).